Sample_v3/Samples/SwiftUISimpleChat/ChannelListView.swift

```diff
@@ -133,7 +133,14 @@ struct ChannelListView: View {
     /// `UserLisView` for users matching the query.
     var userListView: some View {
         let controller = channelList.controller.client.userListController()
-        return UserListView(userList: controller.observableObject)
+        return UserListView(
+            userList: controller.observableObject,
+            didSelectUser: { userId in
+                createDirectMessageChannel(with: userId) {
+                    activeSheet = nil
+                }
```

Comment:
I don't understand this. We create the controller but we don't use it?

Reply:
`createDirectMessageChannel` creates a new DM channel and calls the completion when it is done. In completion we just hide the action sheet. The DM `controller` is created and synchronized inside `createDirectMessageChannel`

Reply:
Sorry, still not understand. Why do we create the channel if we don't use it? There's no need to create the channel upfront. The channel controller should create it once it's presented.